editor.planx.uk/src/@planx/components/TextInput/Public.tsx

```diff
@@ -33,8 +33,9 @@ const TextInputComponent: React.FC<Props> = (props) => {
   });
 
   const characterCountLimit = props.type && getTextLimit(props.type);
-  const displayCharacterCount =
-    Boolean(props.type !== TextInputType.Short && characterCountLimit);
+  const displayCharacterCount = Boolean(
+    props.type !== TextInputType.Short && characterCountLimit,
+  );
```

Comment:
These are linting changes picked up on rebase to passing API tests - sorry for extra noise !